library/std/src/sys/windows/thread.rs

```diff
@@ -101,15 +101,13 @@ impl Thread {
 
 pub fn available_parallelism() -> io::Result<NonZeroUsize> {
     let res = unsafe {
-        let mut sysinfo: c::SYSTEM_INFO = crate::mem::zeroed();
-        c::GetSystemInfo(&mut sysinfo);
-        sysinfo.dwNumberOfProcessors as usize
+        // FIXME: windows::Win32::System::SystemServices::ALL_PROCESSOR_GROUPS should be u16, not u32
```

Comment:
https://github.com/microsoft/win32metadata/commit/8f70180aa6d044c416cd802c36c7d612cf2def98
will be changed to `Windows.Win32.System.Threading.Apis.ALL_PROCESSOR_GROUPS`

Reply:
Hm... I've been thinking about this some more. Could you move the correct definition of `ALL_PROCESSOR_GROUPS` into `c.rs`? That way we can keep all the `FIXME`s into one file so they're easier to find and fix when the bindings are next updated. Otherwise it might be easier to forget about it.